rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/integration/ComponentTest.java

```diff
@@ -401,4 +418,475 @@ public void should_merge_component_with_failure_handling() throws IOException {
 
         assertEquals(HttpStatus.BAD_REQUEST, response.getStatusCode());
     }
+
+    @Test
+    public void should_create_component() throws Exception {
+        Map<String, String> newComponent = new HashMap<>();
+        newComponent.put("name", "Component name");
+        newComponent.put("description", "Component description");
+        newComponent.put("componentType", "OSS");
+        newComponent.put("homepage", "https://spring.io");
+
+        Mockito.doReturn(component).when(componentServiceMock)
+                .createComponent(any(), any());
+
+        HttpHeaders headers = getHeaders(port);
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        ResponseEntity<String> response =
+                new TestRestTemplate().exchange("http://localhost:" + port + "/api/components",
+                        HttpMethod.POST,
+                        new HttpEntity<>(newComponent, headers),
+                        String.class);
+
+        assertEquals(HttpStatus.CREATED, response.getStatusCode());
+        JsonNode responseNode = new ObjectMapper().readTree(response.getBody());
+        assertEquals(component.getName(), responseNode.get("name").textValue());
+        assertEquals(component.getId(), responseNode.get("id").textValue());
+        assertThat(responseNode.get("_embedded").get("createdBy").get("email").textValue(), is(component.getCreatedBy()));
```

Comment:
`assertThat` is deprecated. Please use
```suggestion
        assertEquals(component.getCreatedBy(), responseNode.get("_embedded").get("createdBy").get("email").textValue());
```

See #3278

Reply:
Done, removed assertThat and replace with assertTrue/False/Equals.